### ISSUE TITLE
When does huobi pro ccxt support fetch_deposit_address?

### ISSUE THREAD

**Issue by coincoin1** [ccxt/ccxt#6660]
MUST READ THIS BEFORE SUBMITTING ISSUES (read the links, then delete this message before submitting):

https://github.com/ccxt/ccxt/blob/master/FAQ
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

Make sure your local version of CCXT is up to date. Check by comparing the output of `ccxt.version` to https://github.com/ccxt/ccxt/blob/master/package.json#L3

- OS: Window 10
- Programming Language version: Python 3.7
- CCXT version: 1.23.72

I want to use fetch_Deposit_Address but it is not supported by Huobipro ccxt.

I want to get my deposit address using API.

May I ask if the fetch_deposit_address function is supported?

```
YOUR CODE TO REPRODUCE THE ISSUE WITHOUT YOUR KEYS

import ccxt
huobi = ccxt.huobipro({
    'apiKey': 'key',
    'secret': 'secret_key',
})
print(huobi.fetch_deposit_address())

```

```
YOUR OUTPUT / ERROR / EXCEPTION IN TEXT, NO SCREENSHOTS

Traceback (most recent call last):
  File "C:/Users/user51/PycharmProjects/untitled/huobipro.py", line 8, in <module>
    print(huobi.fetch_deposit_address())
AttributeError: 'huobipro' object has no attribute 'fetch_deposit_address'
```

**Comment by kroitor** [comment]
@coincoin1 we will add it asap! Will let you know when it's there. Thank you!

**Comment by kroitor** [comment]
@coincoin1 good news!

> When does huobi pro ccxt support fetch_deposit_address?

The `huobipro` implementation now supports `fetchDepositAddress()` / `fetch_deposit_address()`, available in CCXT version 1.32.88+. The new build will arrive in 15 minutes.

Let us know if you have any issues with it. Thanks for your feedback!

**Comment by petrkl** [comment]
Hi,
I'm trying to get the deposit address for some currency on the Huobi exchange and I'm having a problem with the fetchDepositAddress() function.
For example, `$exchange->fetchDepositAddress("USDT")` returns an array with an address for only one network.
I tried to manually create some addresses on huobi website but nothing changed in fetchDepositAddress return function.

1. How do I get the deposits address for all available network for a certain currency?
2. When requested (for example Huobi) does this function only show addresses if they are created, or should it also request new addresses?

Thanks for reply!
